### PR TITLE
Add image slider to events

### PR DESCRIPTION
**Pull request by TheFlash98**

### CODE REVIEW

File: _site/soc_projects/10-fossee-scilab-julia.html [hunk 1]
```diff
@@ -255,10 +255,12 @@ <h2 class="post-title" >Scilab-Julia Interface</h2>
 <!--break-->
 
 <h3 id="applicant-profile">Applicant profile:</h3>
-<p>The applicant should-
```
Comment (nihal111) [inline _site/soc_projects/10-fossee-scilab-julia.html]: why did this happen?

Comment (TheFlash98) [inline _site/soc_projects/10-fossee-scilab-julia.html]: What's wrong with it? [see](http://wncc-iitb.org/soc_projects/10-fossee-scilab-julia.html)

Comment (nihal111) [inline _site/soc_projects/10-fossee-scilab-julia.html]: This shouldn't have happened.
Messes up the formatting in that page.
Had to fix it manually.

Comment (TheFlash98) [inline _site/soc_projects/10-fossee-scilab-julia.html]: Okay, okay. Sorry. Will be careful next time.

**Pull request merged**